GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -784,6 +784,7 @@ extension CameraViewController {
         }
         
         alertViewController.addAction(UIAlertAction(title: "Dokumente", style: .default) { [unowned self] _ in
+            self.documentPickerCoordinator.isPDFSelectionAllowed = self.multipageReviewButton.image(for: .normal) == nil
```

Comment:
I would move `self.multipageReviewButton.image(for: .normal) == nil` into a dedicated method for clarity.